README.md

```diff
@@ -63,6 +77,109 @@ Scrape WebMD for drug reviews.
 **-i** Required. Input URL.
 
 ### Example
+
+After installing medinify, run:
+
+```bash
+from medinify.scrapers import WebMDScraper
+
+def main():
+
+    input_url = "https://www.webmd.com/drugs/drugreview-1701-citalopram-oral.aspx?drugid=1701&drugname=citalopram-oral"
+    scraper = WebMDScraper("citalopram_train.csv")
+    scraper.scrape(input_url, 10)
+
+if __name__ == "__main__":
+    main()
+```
+
+Which extracts text reviews from the input URL and outputs them to
+
+ ```bash 
+ citalopram_train.csv
+ ```
+
+## Contributions
```

Comment:
This whole section looks pretty good but I don't know if we're accepting PRs from external contributors at this time. Can you change it so it's directions for just cloning the repo directly instead of forking it?